src/EnergyPlus/PollutionModule.cc

```diff
@@ -758,87 +758,87 @@ namespace PollutionModule {
                     Pollution.NatGasCoef.Source = rNumericArgs(2);
                     if (!lAlphaFieldBlanks(3)) {
                         CheckFFSchedule(
-                            cCurrentModuleObject, "Natural Gas", cAlphaFieldNames(3), cAlphaArgs(3), Pollution.NatGasCoef.SourceSched, ErrorsFound);
+                            cCurrentModuleObject, "NaturalGas", cAlphaFieldNames(3), cAlphaArgs(3), Pollution.NatGasCoef.SourceSched, ErrorsFound);
```

Comment:
For later - in GetPollutionFactorInput - line 1116 - remove synonyms from if statement.